### ISSUE TITLE
opennms missing dependancies - ubuntu 18.04 LTS

### ISSUE THREAD

**Issue by thufunkymunky** [opennms-forge/opennms-install#14]
Hi, I am trying to install opennms on a fresh new install of Ubuntu 18.04 LTS. I am running into the following issues that i am unable to resolve. 

Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 opennms : Depends: opennms-server (= 22.0.3-1) but it is not going to be installed
           Depends: opennms-webapp-jetty (= 22.0.3-1) but it is not going to be installed
           Recommends: opennms-source (= 22.0.3-1) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
FAILED

**Comment by indigo423** [comment]
Can we move this topic to #15?

**Comment by indigo423** [comment]
I was able to reproduce the problem. There is a problem when installing Ubuntu 18.04.1 LTS from a live CD. There is a inconsistent behaviour with the by default installed Ubuntu repositories which are documented here: https://issues.opennms.org/browse/NMS-10503.

In my setup the universe repository was not enabled and I had to enable it manually with the following command:

```
add-apt-repository universe
```

After enabling the universe repository I was able to install opennms correctly.
